dataland-frontend/src/utils/GenericFrameworkTypes.ts

```diff
@@ -33,13 +33,13 @@ export interface Field {
   name: string;
   label: string;
   description: string;
+  unit?: string;
```

Comment:
Why has the order changed here?

Reply:
To be in line with the order in the datamodel.ts files

Reply:
Ok 👍